### PR TITLE
Multus-CNI

### PR DESCRIPTION
**Pull request by sergio-chainguard**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ x ] REQUIRED - The version of the package is still receiving security updates
- [ x ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ x ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: multus-cni.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go-1.21
+      - util-linux
+  environment:
+    CGO_ENABLED: "1"
```
Comment (developer-guy) [inline multus-cni.yaml:18]: is this required? is this binary should be dynamically linked?

File: multus-cni.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  - uses: go/build
+    with:
+      packages: ./cmd/thin_entrypoint
+      ldflags: -X 'gopkg.in/k8snetworkplumbingwg/multus-cni.v4/pkg/multus.version=${{package.version}}'
+      output: thin_entrypoint
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.17.0 google.golang.org/grpc@v1.56.3 google.golang.org/protobuf@v1.33.0
+      go-version: "1.21"
```
[the remark this answers is not in the document]
Comment (sergio-chainguard) [reply multus-cni.yaml]: Awesome, thanks

File: multus-cni.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+  - runs: |
+      set -x
+      mkdir -p /usr/src/multus-cni
+      cp -r . /usr/src/multus-cni
+      ls /usr/src/multus-cni/cmd
```
Comment (ajayk) [inline multus-cni.yaml:32]:
```suggestion
```

File: multus-cni.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  - uses: git-checkout
+    with:
+      repository: https://github.com/k8snetworkplumbingwg/multus-cni
+      tag: v${{package.version}}
+      expected-commit: f03765681fe81ee1e0633ee1734bf48ab3bccf2b
+
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.17.0 google.golang.org/grpc@v1.56.3 google.golang.org/protobuf@v1.33.0
+      go-version: "1.21"
```
Comment (ajayk) [inline multus-cni.yaml:26]:
```suggestion
```

**Review by ajayk — CHANGES_REQUESTED** [review]
Please add some basic tests

**Review by ajayk — APPROVED** [review]

**Pull request merged**